### ISSUE TITLE
Error transcribing file: DecodingOptions.__init__() got an unexpected keyword argument 'progress_callback'

### ISSUE THREAD

**Issue by AnatoliChe** [jhj0517/Whisper-WebUI#66]
Hi!
I have error:
Error transcribing file: DecodingOptions.__init__() got an unexpected keyword argument 'progress_callback'
when use --disable_faster_whisper=true with all models.

Without  --disable_faster_whisper=true it works.

Debian 12.2

**Comment by jhj0517** [comment]
I have not been able to reproduce the error with Windows 10. it seems like a bug with Debians, probably.

**Comment by shuiqingliu** [comment]
same problem with ```-disable_faster_whisper=true```

**Comment by jhj0517** [comment]
You're right. Installing the official whisper will be problematic and I don't think this is good either..
All I did was adding a callback to `trabscribe()` to track the progress in the Gradio UI. 
I was hoping that the official Whisper would provide a generator for this as I made a [PR](https://github.com/openai/whisper/pull/1025) here, but it appears not to.
Perhaps just removing the progress tracking would be proper.

**Comment by mkhadivi** [comment]
Hi, 
I have solved our need for this by adding call_back method.
You can check it [here](https://github.com/mkhadivi/whisper)
